.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
Should we rename this file to `setup.cfg`? This way we can extend it if we introduce additional tools.

Reply:
Agreed and done.